locales/en.yml

```diff
@@ -1,5 +1,5 @@
 en:
   theme_metadata:
     settings:
-      tag_icon_list: 'Enter comma-delimited configuration for tags, in the format "tag-slug,icon,iconColor". Icon color is optional.'
+      tag_icon_list: 'Enter comma-delimited configuration for tags, in the format "tag-slug,icon,color,backgroundColor". The colors are optional.'
```

Comment:
Maybe we should keep the "iconColor" as well in case people only want to change the color of the icon, but others might want to change the color of the text and/or the color of the background?